presto-main/src/test/java/io/prestosql/execution/TestParameterExtractor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.execution;
+
+import io.prestosql.sql.parser.ParsingOptions;
+import io.prestosql.sql.parser.SqlParser;
+import io.prestosql.sql.tree.Parameter;
+import io.prestosql.sql.tree.Statement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestParameterExtractor
+{
+    private final SqlParser sqlParser = new SqlParser();
+
+    @Test
+    public void testNoParameter()
+    {
+        Statement statement = sqlParser.createStatement("SELECT c1, c2 FROM test_table WHERE c1 = 1 AND c2 > 2", new ParsingOptions());
+        assertEqualParameterCount(statement);
+    }
+
+    @Test
+    public void testParameterCount()
+    {
+        Statement statement = sqlParser.createStatement("SELECT c1, c2 FROM test_table WHERE c1 = ? AND c2 > ?", new ParsingOptions());
+        assertEqualParameterCount(statement);
+    }
+
+    private void assertEqualParameterCount(Statement statement)
```

Comment:
Can you explicitly specify expected number of parameters. 0 for the first test and 2 for the second?
The rename this method to `assertParameterCount`